resources/views/users/show.blade.php

```diff
@@ -55,20 +56,20 @@
                                         {{ $competition->id }}
                                     </a>
                                 </td>
-                                <td class="table__cell">
-                                    @if ($competition->reportback)
-                                        <a href="{{ $competition->reportback->admin_url }}">
-                                            {{ $competition->reportback->id }}
-                                        </a>
-                                    @else
-                                        N/A
-                                    @endif
-                                </td>
-                                <td class="table__cell">{{ $competition->reportback->reportback_items->total or 'N/A'}}</td>
-                                <td class="table__cell">{{ $competition->reportback->quantity or 'N/A'}}</td>
-                                <td class="table__cell">{{ $competition->reportback->updated_at or 'N/A'}}</td>
-                                <td class="table__cell">{{ $competition->reportback->flagged or 'N/A' }}</td>
-                                <td class="table__cell"><a href="{{ route('competitions.users.destroy', ['user' => $user->id, 'competition' =>$competition->id]) }}" class="button -danger">Remove</a></td>
+
+                                @if ($competition->reportback)
+                                    <td class="table__cell"><a href="{{ reportback_admin_url($competition->reportback->id) }}">{{ $competition->reportback->id }}</a></td>
+                                    <td class="table__cell">{{ $competition->reportback->reportback_items->total }}</td>
+                                    <td class="table__cell">{{ $competition->reportback->quantity }}</td>
+                                    <td class="table__cell">{{ format_timestamp_for_display($competition->reportback->updated_at) }}</td>
+                                @else
+                                    <td class="table__cell">N/A</td>
```

Comment:
Not really because it needs to check for nested properties of the reportback object if the `$competition->reportback` object exists. There's no way to easily use `or` if it's not the direct item you're looking for.

So say if I did this:
```php
$competition->reportback->quantity or 'NA'
```

If `$competition->reportback` was `null` to begin with I believe it would fail cause then it's trying to get a `quantity` property on `null`.